src/sections/Community/Handbook/Handbook.style.js

```diff
@@ -26,25 +35,19 @@ export const HandbookWrapper = styled.div`
       }
     }
 
+    .content > a:first-of-type > h2:first-of-type {
+      padding-top: 1rem;
+    }
+
+    .highlight{
+      font-weight: 600;
+    }
+
     h2 h3{
       margin: 0.5rem 0;
       color: ${(props) => props.theme.tertiaryColor};
     }
 
-    .page-section{
-      h2{
-        padding-top: 7rem;
-        margin-top: -7rem;
-      }
-      h3{
-        padding-top: 7rem;
-        margin-top: -7rem;
-      }
-      margin-top: -36rem;
-      margin-left: 20rem;
-      display: flex;
-    }
```

Comment:
Apart from these lines there are no actual changes in this PR. Please revert the other changes and limit the changes only to those that are needed.